### DISCUSSION TITLE
API LinkToAnotherRecord Failed with No Error [Mar 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5281] [Discord Threads]
> 🙋‍♂️ **sbx**

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Tagging <@&682859083799199814>  
#  
> 🙋‍♂️ **kyleecodes**  
#### Failing with no error: https://discord.com/channels/661905455894888490/661905455894888493/1079669660590559243  
#  
> 👨‍💻 **pranavxc**  
#### You can use the following APIs to add/remove the value in the LTAR column   
####    
#### https://apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-nested-add   
####    
#### https://apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-nested-remove   
####    
#### You can only link one record at a time using the API, if you want to link multiple then you have to invoke multiple times.   
####    
#### To link you have pass both row ids as path param  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1079476636111622164)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
